### ISSUE TITLE
outdated list of declarable elements in chapter CC

### ISSUE THREAD

**Issue by peterstadler** [TEIC/TEI#1355]
The list under http://www.tei-c.org/release/doc/tei-p5-doc/en/html/CC.html#CCAS2 looks outdated. There are more elements member of att.declarable than are listed there.

**Comment by jamescummings** [comment]
True and uncontroversial in my opinion.  The missing elements are: 

correspDesc equipment geoDecl listApp listEvent listNym listOrg listPerson listPlace metDecl normalization particDesc refsDecl  settingDesc styleDefDecl

However, isn't this a place where we should use one of our nifty TEI processing instructions (like we do for the number of elements somewhere in chapter 1 I think)?  i.e. we want a specList of all the elements of a particular class.  That would be useful to be able to pull in elsewhere.

**Comment by martindholmes** [comment]
I think James's suggestion should be a second ticket. We can exemplify editing the Guidelines initially to fix the immediate problem, then raise another ticket for creating a processing instruction (which is a bit less straightforward and perhaps harder for people to understand).


Cheers,

Martin


Martin Holmes
UVic Humanities Computing and Media Centre
mholmes@uvic.ca
________________________________

**Comment by lb42** [comment]
I wonder whether <specList  key="model.foo"/> ought to do this?

**Comment by jamescummings** [comment]
I like @lb42's suggestion.  Though I wonder if it should be `<specDesc>`, as with other things.  So that would enable having a `<specList>` which mixes multiple classes and elements.  e.g.: (while I don't know why you'd want this particular combination...)
~~~~
<specList>
  <specDesc key="model.declarable"/>
  <specDesc key="persName"/>
  <specDesc key="att.responsibility"/>
</specList>
~~~~

The question is what does the last `<specDesc>` produce?

**Comment by lb42** [comment]
It ought to produce a list of specDescs for the attributes that class provides

**Comment by hcayless** [comment]
Fixed with 192ad75bb3626110e0736efe8bcc73a09e7c9b7c

See TEIC/Stylesheets#127
